### ISSUE TITLE
Default Preview settings

### ISSUE THREAD

**Issue by johnsen32** [ranger/ranger#1438]
#### Runtime Environment

- Operating system and version: Debian Buster rc (testing)
- Terminal emulator and version: gnome-terminal
- Python version: 3.7.2
- Ranger version/commit: 1.9.2
- Locale: us

#### Issue

The ranger manpage states in the Preview section:

`By default, on test files are previewed, but you can enable external preview scripts by setting the option "use_preview_script" and "preview_files" to true.`

However this is not the case. The default setting is already setting these two options to true.
Either this setting is a bug and needs to be changed or this manpage entry is wrong and should be corrected.

**Comment by toonn** [comment]
You're absolutely right! And it's been this way for 6 years : )
Feel free to PR a change to the documentation. (Unless @Vifon thinks we should change the default config to `use_preview_script False`?)

**Comment by Vifon** [comment]
Nah, I think we're fine with this behavior, especially considering it gets hard-disabled for root either way. Let's just fix the docs.
